src/System.CommandLine/Option.cs

```diff
@@ -15,12 +15,29 @@ public class Option :
         private string? _implicitName;
         private protected readonly HashSet<string> _unprefixedAliases = new HashSet<string>();
 
-        public Option(string alias, string? description = null)
-            : this(new[] { alias }, description)
-        {
-        }
-
-        public Option(string[] aliases, string? description = null) : base(description)
+        public Option(
+            string alias,
+            string? description = null,
+            Type? argumentType = null,
+            Func<object?>? getDefaultValue = null,
+            IArgumentArity? arity = null)
```

Comment:
Should this parameter be called `argumentArity`? It's more aligned with e.g. `argumentType`. 

Or should we go the other way and remove the term "argument" from the public interface of `Option`? Is it confusing?